lib/access/oai.rb

```diff
@@ -417,5 +421,5 @@ class EscholProvider < OAI::Provider::Base
   source_model EscholModel.new
   register_format OclcDublinCore.instance
   register_format Marc21.instance
-  update_granularity OAI::Const::Granularity::LOW
+  update_granularity OAI::Const::Granularity::HIGH
```

Comment:
What is the effect of the granularity?

Reply:
There is a field "granularity" in the identify verb of OAI: http://escholarship.org/oai?verb=Identify

@alainna notes in the card that a validator she was using was complaining about the discrepancy between the stated granularity and date reported in the "earliestDatestamp" field.